site/viamguides/src/sticker-wizard/sticker-wizard.md

```diff
@@ -0,0 +1,191 @@
+
+### Prerequisites
+
+- Sign up for a free Viam account, and then [sign in](https://app.viam.com).
+- Hardware and supplies requirements
+  - Raspberry Pi
+  - x3 black acrylic sheets (add dimensions, link)
+  - x3 packs of [craft wire](https://www.amazon.com/TecUnite-Aluminum-Bendable-Skeleton-Thickness/dp/B08GCPWVSZ/)
+  - x9 [360° micro servo motors (we used MG90S)](https://www.amazon.com/Compatible-Raspberry-Project-Helicopter-Airplane/dp/B0925TDT2D?th=1)
+  - x1 [16 channel 12-Bit PWM servo motor driver (PCA9685)](https://www.amazon.com/HiLetgo-PCA9685-Channel-12-Bit-Arduino/dp/B07BRS249H)
+  - x2 5V power supply
+- Software
+  - Install the Typescript SDK on your computer.
+
+### What You’ll Learn
+
+- how to be sad
+
+### What You’ll Need
+
+- A computer running Mac, Windows, or Linux
+- [VS Code](https://code.visualstudio.com/download) installed, or your preferred code editor
+- [Python3](https://www.python.org/downloads/) installed
+- Tools
+  - A computer
+  - Laser cutter
+  - 3D printer
+  - Glue / screwdriver
+  - Wire cutter
+
+### What You’ll Build
+
+- A vending machine controlled by a web application.
+
+<!-- ------------------------ -->
+
+## Set up the electronics
+
+### Set up your Raspberry Pi
+Follow the Viam [documentation](https://docs.viam.com/installation/prepare/rpi-setup/) to set up your Raspberry Pi with the Viam RDK.
+
+### Set up the dispensing mechanisms
+1. Wind up your craft wire into 9 coils of equal size 
+    - Our coils are about 20 loops long and have a diameter just under 2". We recommend using a PVC pipe or otherwise cylindrical object as a guide!
+
+<div>
+  <img width="500" src="assets/coil.jpeg">
+</div>
+
+2. Attach the coils to the motors
+    - We simply wrapped the wire around the double servo arms that come with the motors.
+3. Wire up the motors to the motor board
+4. Wire up the motor board to a 5V power supply
+5. Wire up the motor board to your Raspberry Pi
+    - Follow these [instructions](https://learn.adafruit.com/adafruit-16-channel-servo-driver-with-raspberry-pi/hooking-it-up)
+
+<div>
+  <img width="500" src="assets/motor-board.jpeg">
+</div>
+
+## Configure your machine with Viam
+1. Create a machine on [app.viam.com](https://app.viam.com/)
+2. Add a PCA9685 board component to your machine configuration.
+    - Configure the `i2c_bus` to `1`
+    - Configure the `i2c_address` as `64` (`0x40`), the default base address for [PCA9685](https://learn.adafruit.com/16-channel-pwm-servo-driver/chaining-drivers#addressing-the-boards-848847)
+
+![motor-board-config](assets/motor-board-config.png)
+
+From here, you should be able to use Viam to control your motors! To test:
+1. Open the test section of the board component
+2. Set the pin to be the pin on the board that you connected your motor to
+3. Set the PWM frequency to whatever your motor specifies
+    - Our motors use a PWM frequency of 50 hz
```

Comment:
Is this the recommended frequency of the MG90S? or any micro-servo?

Reply:
this is for the MG90S specifically, not sure about other motors. I've updated